Homework7/Randomizer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework7
+{
+    class Randomizer
```

Comment:
1. `public`
2. Here should be interface because as for now it's not testable

Reply:
fixed